allvers/2004-11-27-GPL-2.0-from-gnu.org.html.notes

```diff
@@ -0,0 +1,4 @@
+Date: 2004-11-27 or earlier
+Link: http://web.archive.org/web/20041127015614/http://www.gnu.org/licenses/gpl.html
+Notes: Beside the HTML format changes, this has the "Temple Place" address like
+ other but the appendix mentions "Lesser" instead of "Library"
```

Comment:
Apart from the format changes this is the exact same text as 2003-12-03 (GNU)

Reply:
Updated in 6d701cb55db6f4f26097400fa36b5b171c165ef1